PopApp/Components/Attendee.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import {StyleSheet, View, Text} from 'react-native'
+
+class Attendee extends React.Component {
```

Comment:
Please add comment, even if its simple like "this class controls and renders Attendee User Interface". Maybe also the high level elements that are (or will be) included in this user interface.